notebooks/proj_sys_eng/sitcom_integration/l3_system_integ/LVV-T2216-plots.ipynb

```diff
@@ -0,0 +1,1166 @@
+    "    num=\"Slew Without LUT Correction - M2Hex Actual Position\", \n",
+    "    dpi=120,\n",
+    "    figsize=(10, 6),\n",
+    "    sharex=True,\n",
+    ")\n",
+    "\n",
+    "cols = \"xyzuvw\"\n",
+    "for ax, col in zip(\"ABCDEF\", cols):\n",
+    "    _ = vandv.hexapod.timeline_position(\n",
+    "        axs[ax], \n",
+    "        [pos, cpos, upos, lut], \n",
+    "        column=col, \n",
+    "        elevation=el, \n",
+    "        symbols=[\"\", \"o\", \"s\", \"\"],\n",
+    "        names=[\"Actual Position\", \"Compensated\", \"Uncompensated\", \"LUT\"]\n",
+    "    )\n",
+    "\n",
+    "# Hide xlabel\n",
+    "for i in \"ABDE\":\n",
+    "    _ = axs[i].set_xlabel(\"\")\n",
+    "    \n",
+    "_ = axs[\"F\"].legend(loc='lower center', bbox_to_anchor=(0.5, -1), ncol=5)\n",
+    "\n",
+    "fig.suptitle(\"M2Hex Timeline - Slew w/ Compensation\")\n",
+    "fig.autofmt_xdate()\n",
+    "fig.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5b51c6dd-ad03-4f15-a871-28409becd258",
+   "metadata": {},
+   "source": [
+    "## Compare Snapshots with LUT\n",
+    "\n",
+    "Compare the average snapshots in different elevations using the LUT."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c41d945d-9b8c-448b-bc73-647fd3da44b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "el = await client.select_time_series(\n",
+    "    \"lsst.sal.MTMount.elevation\",\n",
+    "    \"*\", \n",
+    "    t_start,\n",
+    "    t_end)\n",
+    "\n",
+    "# Elevation is 85.4 deg\n",
+    "el85 = el[np.isclose(el.actualPosition, 85.4)]\n",
+    "t_start85 = el85.index[0]\n",
+    "t_end85 = el85.index[-1]\n",
+    "\n",
+    "# Elevation is 84.4 deg\n",
+    "el84 = el[np.isclose(el.actualPosition, 84.4)]\n",
+    "t_start84 = el84.index[0]\n",
+    "t_end84 = el84.index[-1]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9fbe54f8-3e33-4b8f-b046-a95d91bf96a2",
+   "metadata": {},
+   "source": [
+    "### Analyse M1M3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1c087fb-3761-4045-8010-4f0c932f9d8a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "series = [df.loc[t_start85:t_end85].mean() - df.loc[t_start84:t_end84].mean() for df in [fel, fba, fst, fad]]\n",
+    "\n",
+    "labels = [\n",
+    "    \"appliedElevationForces\", \n",
+    "    \"appliedBalanceForces\", \n",
+    "    \"appliedStaticForces\",\n",
+    "    \"forceActuatorData\",\n",
+    "]\n",
+    "\n",
+    "fig, axs = plt.subplot_mosaic(\n",
+    "    mosaic=\"AB\\nAB\\nCC\\nCC\",\n",
+    "    num=\"Slew Without Correction - Snapshot Force Differences - 85.4 and 84.4\", \n",
+    "    constrained_layout=True,\n",
+    "    dpi=120,\n",
+    "    figsize=(9, 5),\n",
+    ")\n",
+    "\n",
+    "fig.suptitle(\"M1M3 Snapshot - LUT+mount - XYZ (85.4 deg - 84.4 deg)\")\n",
+    "_ = vandv.m1m3.snapshot_xforces(axs[\"A\"], series)\n",
```

Comment:
Keep the same plot layout as in previous snapshots, with, the top row showing "forceActuatorData" and "AppliedElevationForces"  and the bottom "AppliedBalanceForces" and "AppliedActiveOpticsForces"

Reply:
Done